ross/rotor_assembly.py

```diff
@@ -3115,7 +3150,7 @@ def __init__(
             dof_tuple = namedtuple("GlobalIndex", global_dof_mapping)
             elm.dof_global_index = dof_tuple(**global_dof_mapping)
             df.at[
-                df.loc[df.tag == elm.tag].index[0], "dof_global_index"
+                df.loc[df.tag == elm.tag].index[0], "dof_globaxl_index"
```

Comment:
Typo here `dof_globaxl_index`, I think.
Make it `dof_global_index`

Reply:
Yup, we missed that one, thanks!